etc/workflows/publish-after-cutting.xml

```diff
@@ -25,6 +25,7 @@
         <configuration key="publishToYouTube">false</configuration>
         <configuration key="thumbnailType">0</configuration>
         <configuration key="thumbnailPosition">1</configuration>
+        <configuration key="flagQuality720p">true</configuration>
```

Comment:
What is the flag quality for?

Reply:
The problem with any event processed by a workflow like fast is that the configuration is different from “Schedule and Upload”. Using “Start Task”, that's no problem since you have a new configuration panel and will set the options again. But that's not true for the editor which relies on previously set options. This means that when starting “Publish” from the editor, no quality-flags are set and no encoding will be done. That ultimately results in no processed media and a failure during publication. That would also be a problem with your previous approach.

This flag sets the same default set in “Schedule and Upload”, meaning that at least one quality will be encoded if nothing else is set. If something else had been set before (e.g. because “Schedule and Upload” had been used), this will have no effect.